lib/salus/scanners/report_ruby_gems.rb

```diff
@@ -76,8 +80,50 @@ def record_ruby_gem(name:, version:, source:, dependency_file:)
         type: 'gem',
         name: name,
         version: version,
-        source: source
+        source: source,
+        licenses: find_licenses_for(name, version)
       )
     end
+
+    def find_licenses_for(gem_name, version)
+      res = Net::HTTP.get_response(ruby_gems_uri_for(gem_name, version))
+
+      return [] unless res.is_a?(Net::HTTPSuccess)
+
+      licenses = JSON.parse(res.body)['licenses']
+      licenses.map { |license| spdx_license_for(license) }
+    rescue StandardError => e
+      msg = "Unable to gather license information " \
+        "using rubygems api" \
+        "with error message #{e.class}: #{e.message}"
+      bugsnag_notify(msg)
+
+      []
+    end
+
+    def ruby_gems_uri_for(gem_name, version)
+      URI("https://rubygems.org/api/v2/rubygems/#{gem_name}/versions/#{version}.json")
```

Comment:
I wonder how common this would cause conflicts, because not all gems are pulled from rubygems.org. 

Some issues:
* Gem A from rubygems.org and random_server.com may have difference license info
* Gem A exists on random_server.com, but not rubygems.org
  * If this is too common, then we will get a large number of bugsnag messages and pager duty may ping us crazy.

Reply:
If we decide to not fix this, then we should clearly communicate this and other issues with the scanner in the
* PR description 
* new salus version release notes.

Reply:
We are focusing license information about external ruby gems which come from rubygems.org. The internal ruby gems come from another source. That's why it would be great if we don't send message to bugsnag for that.

Reply:
Updated this part. Thanks.